python/tank/platform/bundle.py

```diff
@@ -963,10 +965,12 @@ def __resolve_hook_path(self, settings_name, hook_expression):
             # possible declaration of correct hook name in the sgtk_config preferences files.
             path = hook_expression.replace("{show}/", "")
```

Comment:
`path = hook_expression.replace("{show}", "SHARED/etc/sgtk/hooks")`

`"etc/sgtk/hooks"` is deprecated and shouldn't be used.

Reply:
Then I will need to add `user_subdirectories="etc/sgtk/hooks"` and `show_subdirectories= "SHARED/etc/sgtk/hooks"` differently.